kuma/users/views.py

```diff
@@ -945,7 +945,7 @@ def _send_payment_received_email(payment_intent, locale):
         "payment_date": datetime.fromtimestamp(payment_intent.created),
         "next_payment_date": subscription_info["next_payment_at"],
         "invoice_number": payment_intent.id,
-        "dollar_amount": settings.CONTRIBUTION_AMOUNT_USD,
+        "cost": settings.CONTRIBUTION_AMOUNT_USD,
```

Comment:
@peterbe I renamed it after all, given that we now have a formatting function that could work with other locales, I'd find the naming misleading. Since the english original doesn't contain the currency symbol, is it too hopeful to assume that the localizers don't introduce one themselves?